lib/codec_choose.c

```diff
@@ -148,21 +157,21 @@ codec_choose_arm (struct codec *codec)
 static bool
 codec_choose_x86 (struct codec *codec)
 {
-#if (__x86_64__ || __i386__ || _M_X86 || _M_X64) && (HAVE_AVX2 || HAVE_SSSE3 || HAVE_SSE41 || HAVE_SSE42)
+#if (__x86_64__ || __i386__ || _M_X86 || _M_X64) && (HAVE_AVX2 || HAVE_SSSE3 || HAVE_SSE41 || HAVE_SSE42 || HAVE_AVX)
```

Comment:
If you change the preprocessor stuff to the format above (or something else), you might as well do it in the SSE4.1 commit straight away.